x-pack/legacy/plugins/uptime/public/pages/certificates.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { useDispatch } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSpacer } from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { useUptimeTelemetry, UptimePage } from '../hooks';
+import { useTrackPageview } from '../../../../../plugins/observability/public';
+import { PageHeader } from './page_header';
+import { useBreadcrumbs } from '../hooks/use_breadcrumbs';
+import { CertificateList, CertSort } from '../components/certificates/certificates_list';
+import { CertificateSearch } from '../components/certificates/certificate_search';
+import { OVERVIEW_ROUTE, SETTINGS_ROUTE } from '../../common/constants';
+import { getDynamicSettings } from '../state/actions/dynamic_settings';
+import { getCertificatesAction } from '../state/certificates/certificates';
+
+export const CertificatesPage: React.FC = () => {
+  useUptimeTelemetry(UptimePage.Certificates);
+
+  useTrackPageview({ app: 'uptime', path: 'certificates' });
+  useTrackPageview({ app: 'uptime', path: 'certificates', delay: 15000 });
+
+  useBreadcrumbs([{ text: 'Certificates' }]);
+
+  const [page, setPage] = useState({ index: 0, size: 10 });
+  const [sort, setSort] = useState<CertSort>({
+    field: 'certificate_not_valid_after',
+    direction: 'asc',
+  });
+  const [search, setSearch] = useState('');
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(getDynamicSettings());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(
+      getCertificatesAction.get({ search, ...page, sortBy: sort.field, direction: sort.direction })
+    );
+  }, [dispatch, page, search, sort.direction, sort.field]);
+
+  return (
+    <>
+      <EuiFlexGroup>
+        <EuiFlexItem>
+          <Link to={OVERVIEW_ROUTE} data-test-subj="uptimeCertificatesToOverviewLink">
```

Comment:
Why is this button wrapped in a link?

Reply:
Well, EUI button can't be attached to react router, so we have to use React Router Link Component as wrap.